packages/ckeditor5-dev-release-tools/tests/utils/executeinparallel.js

```diff
@@ -0,0 +1,550 @@
+			expect( firstWorker.workerData ).to.have.property( 'packages' );
+
+			expect( secondWorker.workerData ).to.be.an( 'object' );
+			expect( secondWorker.workerData ).to.have.property( 'callbackModule', '/home/ckeditor/uuid-4.js' );
+			expect( secondWorker.workerData ).to.have.property( 'packages' );
+
+			// Workers did not emit an error.
+			getExitCallback( firstWorker )( 0 );
+			getExitCallback( secondWorker )( 0 );
+
+			await promise;
+		} );
+
+		it( 'should use the specified `cwd` when looking for packages', async () => {
+			const options = Object.assign( {}, defaultOptions, {
+				cwd: '/custom/cwd'
+			} );
+
+			const promise = executeInParallel( options );
+
+			expect( stubs.glob.globSync.callCount ).to.equal( 1 );
+			expect( stubs.glob.globSync.firstCall.args[ 0 ] ).to.equal( 'my-packages/*/' );
+			expect( stubs.glob.globSync.firstCall.args[ 1 ] ).to.be.an( 'object' );
+			expect( stubs.glob.globSync.firstCall.args[ 1 ] ).to.have.property( 'cwd', '/custom/cwd' );
+
+			const [ firstWorker, secondWorker ] = WorkerMock.instances;
+
+			// Workers did not emit an error.
+			getExitCallback( firstWorker )( 0 );
+			getExitCallback( secondWorker )( 0 );
+
+			await promise;
+		} );
+
+		it( 'should normalize the current working directory to unix-style (default value, Windows path)', async () => {
+			stubs.process.cwd.returns( 'C:\\Users\\ckeditor' );
+			const promise = executeInParallel( defaultOptions );
+
+			expect( stubs.glob.globSync.callCount ).to.equal( 1 );
+			expect( stubs.glob.globSync.firstCall.args[ 0 ] ).to.equal( 'my-packages/*/' );
+			expect( stubs.glob.globSync.firstCall.args[ 1 ] ).to.be.an( 'object' );
+			expect( stubs.glob.globSync.firstCall.args[ 1 ] ).to.have.property( 'cwd', 'C:/Users/ckeditor' );
+
+			const [ firstWorker, secondWorker ] = WorkerMock.instances;
+
+			// Workers did not emit an error.
+			getExitCallback( firstWorker )( 0 );
+			getExitCallback( secondWorker )( 0 );
+
+			await promise;
+		} );
+
+		it( 'should normalize the current working directory to unix-style (`options.cwd`, Windows path)', async () => {
+			const options = Object.assign( {}, defaultOptions, {
+				cwd: 'C:\\Users\\ckeditor'
+			} );
+
+			const promise = executeInParallel( options );
+
+			expect( stubs.glob.globSync.callCount ).to.equal( 1 );
+			expect( stubs.glob.globSync.firstCall.args[ 0 ] ).to.equal( 'my-packages/*/' );
+			expect( stubs.glob.globSync.firstCall.args[ 1 ] ).to.be.an( 'object' );
+			expect( stubs.glob.globSync.firstCall.args[ 1 ] ).to.have.property( 'cwd', 'C:/Users/ckeditor' );
+
+			const [ firstWorker, secondWorker ] = WorkerMock.instances;
+
+			// Workers did not emit an error.
+			getExitCallback( firstWorker )( 0 );
+			getExitCallback( secondWorker )( 0 );
+
+			await promise;
+		} );
+
+		it( 'should create the temporary module properly when using Windows-style paths', async () => {
+			stubs.process.cwd.returns( 'C:\\Users\\ckeditor' );
```

Comment:
*minor*

I don't think we should test the OS specific paths in this place.
This way such tests should be present in all new release functions.

Repeating such tests causes the test file to grow in size when it's not needed.

Reply:
This test aims to confirm if the normalized `process.cwd()` path is used when storing the temporary module. Not sure if needed. I prefer to add more than less.